searchEngine/MockArticle.py

```diff
@@ -0,0 +1,16 @@
+class Article:
+    """
+    Has all the attributes that an article table will have in the db
+    Attributes:
+        id: given id in db
+        number: article number extracted from the pdf
+        content: article's full text
+        keywords: frequency map of the keywords from the content
```

Comment:
Nice! I guess we are both going to use this class. I'm still not sure about the folder structure but maybe we should put it one level up?

Reply:
Yeah, probably both of your services could be grouped in a subfolder of the project.

Reply:
moved it